db/libdwarf

```diff
@@ -0,0 +1,15 @@
+R2PM_BEGIN
+
+R2PM_GIT "git://git.code.sf.net/p/libdwarf/code" libdwarf-code
+R2PM_DESC "install libdwarf"
+
+R2PM_INSTALL() {
+	./configure --prefix="${R2PM_PREFIX}"
+	${MAKE} PREFIX="${R2PM_PREFIX}"
```

Comment:
this is not installing libdwarf, right? why is PREFIX needed to be passed again? isnt inferred by the configure?